picky-server/src/db/file.rs

```diff
@@ -255,6 +273,28 @@ impl PickyStorage for FileStorage {
         .boxed()
     }
 
+    fn increase_issued_authenticode_timestamps_counter(&self) -> BoxFuture<'_, Result<(), StorageError>> {
+        let name = format!("{}{}", "timestamp_counter_store", TXT_EXT).replace(" ", "_");
```

Comment:
You don’t need the `replace` here (copy paste mistake?)

Reply:
> copy-paste mistake?

Exactly :slightly_smiling_face: